pkg/scope/provider.go

```diff
@@ -153,14 +163,25 @@ func NewProviderClient(cloud clientconfig.Cloud, caCert []byte, logger logr.Logg
 	}
 
 	config := &tls.Config{
-		RootCAs:    x509.NewCertPool(),
 		MinVersion: tls.VersionTLS12,
 	}
 	if cloud.Verify != nil {
 		config.InsecureSkipVerify = !*cloud.Verify
 	}
-	if caCert != nil {
-		config.RootCAs.AppendCertsFromPEM(caCert)
+
+	if useSystemRootCAs {
+		roots, err := x509.SystemCertPool()
```

Comment:
Just a note about this block.  I did initially try without, and I thought it would just pick them up implicitly but it didn't, that is unless I made a mistake at some point, which is highly likely considering how complex this entire ecosystem is.  Besides this provides additional error checking, so never a bad thing.